test/activity.js

```diff
@@ -28,14 +28,14 @@ describe('Activity', () => {
     )
     await fs.writeFile(
       getPaths(CACHE_ROOT, STATE_ROOT).activity,
-      '[3/14/2023, 10:38:14 AM] {"source":"Saturn","type":"info","message":"beep boop"}\n'
+      '[2023-04-26T12:26:40.489Z] {"source":"Saturn","type":"info","message":"beep boop"}\n'
     )
     const { stdout } = await execa(
       station,
       ['activity'],
       { env: { CACHE_ROOT, STATE_ROOT } }
     )
-    assert.match(stdout, /3\/14\/2023/)
+    assert.match(stdout, /2023/)
```

Comment:
I guess this is good enough.

An alternative I used while trying to fix the failing tests - I am posting it just for consideration.

```suggestion
    assert.match(stdout, /(3\/14\/2023)|(14\/03\/2023)/)
```

Reply:
Then the next person changes their locale to German or something else and it will break again

Reply:
I think there are only a handful of different ways to format a date, so I would not worry about scalability.

Anyhow, I am fine with checking just the year.